pydantic_evals/pydantic_evals/evaluators/evaluator.py

```diff
@@ -146,17 +146,38 @@ def evaluate(self, ctx: EvaluatorContext) -> bool:
     __pydantic_config__ = ConfigDict(arbitrary_types_allowed=True)
 
     @classmethod
-    def name(cls) -> str:
+    def get_serialization_name(cls) -> str:
         """Return the 'name' of this Evaluator to use during serialization.
 
         Returns:
             The name of the Evaluator, which is typically the class name.
         """
-        # Note: if we wanted to prefer snake_case, we could use:
-        # from pydantic.alias_generators import to_snake
-        # return to_snake(cls.__name__)
         return cls.__name__
 
+    @classmethod
+    @deprecated('`name` has been renamed, use `get_serialization_name` instead.')
+    def name(cls) -> str:
+        """`name` has been renamed, use `get_serialization_name` instead."""
+        return cls.get_serialization_name()
+
+    def get_default_evaluation_name(self) -> str:
```

Comment:
What do you think about calling this `label`, so we can keep the other one as `name`?